be1-go/channel/federation/federation.go

```diff
@@ -40,15 +42,17 @@ const (
 )
 
 type remoteOrganization struct {
-	laoId       string
-	fedChannel  string
-	organizerPk string
+	laoId             string
+	fedChannel        string
+	organizerPk       string
+	signedOrganizerPk string
```

Comment:
The original purpose of that field is that, in the result message the server must be able to verify the signature on the challenge (this is done because the challenge is a standalone-message signed by the remote organizer) and also verify a signature on the pk of the local organizer, this is however not happening right now---this is an oversight, i'll change asap.